spec/tree_spec.rb

```diff
@@ -1,14 +1,79 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+describe Tree do
+  before(:all) do
+    @my_tree = Tree.new
+  end  
+
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(@my_tree.is_a? Tree).to be true
```

Comment:
RSpec has a ton of features (too many?) to make your specs read well. This one could be:

```ruby
expect(@my_tree).to be_a(Tree)
```